DemocracyClub/WhoCanIVoteFor
# Feat/party pages descriptions

Next phase of the Party Pages overhaul - now featuring more useful data for voters! [Trello](https://trello.com/c/dVLjcV1A/3225-wcivf-political-party-pages).

This update adds:
- Party descriptions, ordered the same way as they're ordered on the EC website (date last change authorised)
- An indicator as to which register the party is on (GB or NI)

Screenshots
<img width="860" alt="Screenshot 2023-04-20 at 14 30 04" src="https://user-images.githubusercontent.com/9531063/233383880-5d61a152-5a04-4143-ba93-721a735188ab.png">
<img width="864" alt="Screenshot 2023-04-20 at 14 29 52" src="https://user-images.githubusercontent.com/9531063/233383889-da7b87c6-8787-4006-95c2-76cc0bf12807.png">

To test locally:
Part 1
- Make sure you're fully up to date with migrations and `import_parties` 
- Run the project
- Go to a party page who has descriptions (I chose `party:52`)
- Check that you can see the descriptions
- Go through to the EC link
- Check that the descriptions are in the correct order

Part 2
- Find a party you know to be registered in GB (`party:52` works here too)
- Check that you can see "This party is on the Great Britain register."

Part 3
- Find a party you know to be registered in NI (I used `party:51` - very convenient)
- Check that you can see "This party is on the Northern Ireland register."

```[tasklist]
### PR Checklist

```
